docs/src/intro.md

```diff
@@ -21,20 +21,20 @@ SpaceShip(Person(:julia, 2009), [0.0, 0.0, 0.0], [0.0, 0.0, 0.0])
 ```
 Lets update the captains name:
 ```jldoctest spaceship
-julia> s.captain.name = "JULIA"
+julia> s.captain.name = :JULIA
```

Comment:
So Julia 1.0 doesn't do the automatic conversion. I have to change it everywhere.